Tasks/kudaka/Task6/Task_6.py

```diff
@@ -0,0 +1,65 @@
+depth_list = [1,2,[3,4,[5,6],8,[2,[3,7,8],5,8],[7,8],9]]
+def sum_list_numb(l:list) -> "taking list return sum(number)":
+    if len(l) == 1:
+        if isinstance(l[0],list):
+            riturn = sum_list_numb(l[0])
+        else:
+            riturn = l[0]
+    elif isinstance(l[0],list):
+        remove = l.pop(1)
+        l[0].append(remove)
+        riturn = sum_list_numb(l)
+    elif isinstance(l[1], list):
+        remove = l.pop(0)
+        l[0].append(remove)
+        riturn = sum_list_numb(l)
+    else:
+        remove = l.pop(1)
+        l[0] += remove
+        riturn = sum_list_numb(l)
+    return riturn
+print(sum_list_numb(depth_list))
```

Comment:
it's overcomplicated, basically the solution should check only two options (is type list or not) and do not alter the list itself